common/setup.py

```diff
@@ -29,9 +22,28 @@
     maintainer='Amundsen TSC',
     maintainer_email='amundsen-tsc@lists.lfai.foundation',
     packages=find_packages(exclude=['tests*']),
-    install_requires=requirements_common,
+    install_requires=[
+        # Packages in here should rarely be pinned. This is because these
```

Comment:
note, I didn't author this comment, I took it from the earlier version of `amundsen-common`